doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -25,15 +25,131 @@
 Traceability to Reference Model.
 
 <a name="2.3"></a>
-## 2.3 Requirements.
+## 2.3 Containerised Platform Requirements.
```

Comment:
Do we keep the dots in headers?

Reply:
I would suggest they are removed from all headers.